crypto-hashes/Cargo.toml

```diff
@@ -9,20 +9,20 @@ repository = "https://github.com/RustCrypto/meta"
 keywords = ["crypto", "meta", "hash", "digest"]
 
 [dependencies]
-blake2 = "0.9"
-gost94 = { version="0.9", optional = true }
-groestl = "0.9"
-md2 = { version = "0.9", optional = true }
-md4 = { version = "0.9", optional = true }
-md-5 = { version = "0.9", optional = true }
+blake2 = "0.10"
+gost94 = { version="0.10", optional = true }
+groestl = "0.10"
+md2 = { version = "0.10", optional = true }
+md4 = { version = "0.10", optional = true }
+md-5 = { version = "0.10", optional = true }
 ripemd160 = "0.7"
-sha-1 = { version = "0.9", optional = true }
-sha2 = "0.9"
-sha3 = "0.9"
-streebog = { version = "0.9", optional = true }
-whirlpool = "0.9"
+sha-1 = { version = "0.10", optional = true }
```

Comment:
You also need to change `ripemd160` to `ripemd` v0.1 and modify the re-export and docs in `lib.rs` accordingly. While we at it, it also may be worth to port this crate to 2018 edition.

Reply:
Thanks for review! I didn;t know what to do about `ripemd160`.